packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -89,14 +87,8 @@ export const BootDataProvider = ({
     useState<Partial<BootCacheData>>();
   const [initialLoad, setInitialLoad] = useState<boolean>(null);
   const loadedFromCache = !!cachedBootData;
-  const {
-    user,
-    settings,
-    flags = {},
-    alerts,
-    notifications,
-    squads,
-  } = cachedBootData || {};
+  const { user, settings, alerts, notifications, squads } =
```

Comment:
With this we need to ensure we also remove it on API side.
But might have to consider rewriting the can_submit_article in that case.

Reply:
We can't remove the flags on the API side yet to support extension users until we get certain amount of adoption rates. Let me create a task for clean up.